readme.md

```diff
@@ -304,7 +304,7 @@ Thanks for contributing! 🦋🙌
 - [](# "tags-on-commits-list") [Displays the corresponding tags next to commits.](https://user-images.githubusercontent.com/14323370/66400400-64ba7280-e9af-11e9-8d6c-07b35afde91f.png)
 - [](# "mark-merge-commits-in-list") [Marks merge commits in commit lists.](https://user-images.githubusercontent.com/16872793/75561016-457eb900-5a14-11ea-95e1-a89e81ee7390.png)
 - [](# "deep-reblame") [When exploring blames, `Alt`-clicking the “Reblame” buttons will extract the associated PR’s commits first, instead of treating the commit a single change.](https://user-images.githubusercontent.com/16872793/77248541-8e3f2180-6c10-11ea-91d4-221ccc0ecebb.png)
-- [](# "highlight-deleted-and-added-files-in-diffs") [Indicates with an icon whether files in commits and pull requests being added or removed.](https://user-images.githubusercontent.com/1402241/90332474-23262b00-dfb5-11ea-9a03-8fd676ea0fdd.png)
+- [](# "new-or-deleted-file") [Indicates with an icon whether files in commits and pull requests being added or removed.](https://user-images.githubusercontent.com/1402241/90332474-23262b00-dfb5-11ea-9a03-8fd676ea0fdd.png)
```

Comment:
New name ok?

Reply:
Hmm, I don't find it as clear as the old one. Maybe `new-or-deleted-file-diff-indicator`?

Reply:
Feature names should no longer be fully descriptive, I'm only renaming long feature names because they're impossible to mention in commits or to remember. See also the other recent renames in `options-storage`

Reply:
See https://github.com/refined-github/refined-github/issues/6707

Reply:
I agree that the new name isn't very descriptive, but between descriptiveness and brevity we should prefer the latter, as described in #6707. 

Alternative short names are welcome; names that are only slightly shorter than what we have aren't enough.

Reply:
`file-diff-type-indicator`?

Reply:
While that sounds correct, it gives me even less information about what the indicator is about. "diff type" == ?

Maybe `mark-new-or-deleted-file` but that's so long too.